### ISSUE TITLE
Release 3.0.0

### ISSUE THREAD

**Issue by monken** [ajv-validator/ajv-formats#75]
Hi, is there a reason for holding off on the 3.0.0 stable release? Many important bug fixes around date-time and the byte format have been committed. Looking forward to an official release!

**Comment by tschmidtb51** [comment]
3.0.0-rc still has a bug - see discussion in #55. However, there is a PR (#82) that should fix the issue... We are happy to receive comments on that.

**Comment by kennethaasan** [comment]
Does this project have active maintainers? It would be great to get https://github.com/ajv-validator/ajv-formats/pull/87 merged into 3.0.0 as well.

**Comment by mustafababil** [comment]
Looks like v3 has arrived a couple of days ago. However, it misses proper release notes.

**Comment by Twipped** [comment]
Yeah, I came here wondering what merited the major version bump. Is this safe to update to, or are there needed code changes?

**Comment by jm42** [comment]
One year later. Still holding the release notes.
